robotbase/src/main/java/com/team2357/frc2022/RobotContainer.java

```diff
@@ -38,21 +38,26 @@
   private final IntakeDriveControls m_driverControls;
   private final GunnerControls m_gunnerControls;
 
-  private final RobotArduino m_arduinoSensor;
+  private final DigitalInput m_intakeSensor;
+  private final DigitalInput m_feederSensor;
+  private final DigitalInput m_turretSensor;
 
   /**
    * The container for the robot. Contains subsystems, OI devices, and commands.
    */
   public RobotContainer() {
-    m_arduinoSensor = new RobotArduino(Constants.ARDUINO.ARDUINO_SENSOR_DEVICE_NAME);
+    m_intakeSensor = new DigitalInput(Constants.SENSORS.INTAKE_SENSOR_DIO_PORT);
+    m_feederSensor = new DigitalInput(Constants.SENSORS.FEEDER_SENSOR_DIO_PORT);
+    m_turretSensor = new DigitalInput(Constants.SENSORS.TURRET_SENSOR_DIO_PORT);
```

Comment:
We can delete the turret sensor stuff out of robot container, since we will no longer have a magnet sensor on the turret.